docs/cloud/azureai/use-flow-in-azure-ml-pipeline.md

```diff
@@ -133,8 +196,50 @@ Above is part of the pipeline job python code, see here for [full example](https
 
 ::::
 
+Like regular parallel components, customer may specify run settings for them in a pipeline job. Some regularly used run settings have been listed in section [Component ports and run settings](#component-ports-and-run-settings); customer may also refer to [the official document of parallel component](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-parallel-job-in-pipeline?view=azureml-api-2&tabs=cliv2) for more details.
+
+## Environment of the component
+
+By default, the environment of the created component will be based on the latest promptflow runtime image. If customer has [specified python requirement file](../../reference/flow-yaml-schema-reference.md) in `flow.dag.yaml`, they will be applied to the environment automatically:
+
+``` yaml
+...
+environment:
+  python_requirements_txt: requirements.txt
```

Comment:
In eager flow we allow auto pickup requirements.txt, even user not set it in flow.dag.yaml. 

Will it work for flow as component scenario?

Reply:
1. it won't work in flow as component scenario
2. actually eage flow would be blocked on component creation given there won't be a `flow.tools.json` according to current logic.

We'll need some extra SDK changes and potential PFS changes to enable eager flow:
1. SDK may handle auto pickup requirements.txt and construct an azure ml environment like we discussed before, which won't require service change;
2. for now, service depends on `flow.dag.yaml` & `flow.tools.json` to extract ports for the component, I doubt that there would be some necessary changes.

I have deleted some related content as eager flow is still private for now.

Reply:
![image](https://github.com/microsoft/promptflow/assets/37076709/05186f6a-4a87-49bd-a681-bbac12eefb78)

Reply:
![image](https://github.com/microsoft/promptflow/assets/37076709/e2738f16-8819-48b7-ad9d-d590fb9dee6e)
after I provide a dummy `flow.tools.json`